deploy/install.sh

```
@@ -0,0 +1,139 @@
+# 下载配置文件
+if [ -f ${bin_dir}/config/config.toml ]; then
+    echo "配置文件已存在, 跳过下载"
+    echo "[WARNING] 请检查配置文件是否正确，DEV版本升级时请注意配置文件兼容性"
+    sleep 2
+else
+    wget -q -O ${bin_dir}/config/config.toml https://raw.githubusercontent.com/WJQSERVER/speedtest-ex/main/config/config.toml
+fi
+
+# 下载systemd服务文件
+if [ "$bin_dir" = "${bin_dir_default}" ]; then
+    make_systemd_service()
```

Shellcheck (0.10.0)

[error] Couldn't parse this function. Fix to allow more checks.

(SC1073)